stable/mlrun/values.yaml

```diff
@@ -605,6 +605,11 @@ httpDB:
   # dsn: "mysql+pymysql://root@mlrun-db:3306/mlrun"
   # oldDsn: "sqlite:////mlrun/db/mlrun.db?check_same_thread=false"
 
+modelMonitoring:
+  # If no dsn has been provided, will initiate monitoring database under mlrun-db service:
+  # mysql+pymysql://root@mlrun-db:3306/mlrun-monitoring
```

Comment:
should be mlrun_monitoring, no?

Reply:
Right, I'll rename it to mlrun_model_monitoring which is indeed clearer than just mlrun_monitoring that can refer to other monitoring aspects as well.